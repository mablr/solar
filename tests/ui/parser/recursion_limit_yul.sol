contract C {
    function f() public {
        assembly {
            {{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}} //~ ERROR: recursion limit reached
        }
   }
}
