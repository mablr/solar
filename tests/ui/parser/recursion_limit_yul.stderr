error: recursion limit reached
  --> ROOT/tests/ui/parser/recursion_limit_yul.sol:LL:CC
   |
LL | ...{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{{}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}...
   |                                                      -^
   |                                                      |
   |                                                      while parsing Yul statement

error: aborting due to 1 previous error

